src/widgetastic/log.py

```diff
@@ -30,8 +30,15 @@ def call_sig(args: Iterator[Any], kwargs: MutableMapping[str, Any]) -> str:
     Returns:
         A string that contains parameters in parentheses like the call to it.
     """
-    arglist = [repr(x) for x in args]
-    arglist.extend(f"{k}={v!r}" for k, v in kwargs.items())
+    safe_args = ["sensitive", "locator"]
```

Comment:
why locator?

Reply:
i suppose debugging is completely ruined unless at least locator is not hidden